packages/main/src/plugin/container-registry.spec.ts

```diff
@@ -410,6 +415,15 @@ vi.mock('node:stream/promises', async () => {
   };
 });
 
+vi.mock(import('node:fs/promises'), async importOriginal => {
+  const actual = await importOriginal();
+  return {
+    ...actual,
+    readFile: vi.fn(),
+    rm: vi.fn(),
+  };
+});
+
```

Comment:
question: hello, I'm wondering why we're mocking half of the method

spying only readFile and rm at the beginning of the unit test is not enough ?

Reply:
I'm not sure what would be the best solution. I pushed a new commit. Is that what you are thinking about?

Reply:
yes there were several choice, you took one, I'm fine